diff --git a/node_modules/@fastify/busboy/lib/main.js b/node_modules/@fastify/busboy/lib/main.js
index 8794beb..de4754f 100644
--- a/node_modules/@fastify/busboy/lib/main.js
+++ b/node_modules/@fastify/busboy/lib/main.js
@@ -1,7 +1,7 @@
 'use strict'
 
-const WritableStream = require('node:stream').Writable
-const { inherits } = require('node:util')
+const WritableStream = require('stream').Writable
+const { inherits } = require('util')
 const Dicer = require('../deps/dicer/lib/Dicer')
 
 const MultipartParser = require('./types/multipart')
